src/docz/Chip.mdx

```diff
@@ -0,0 +1,52 @@
+---
+name: Chip
+route: /chip
+---
+
+import Avatar from '../components/Avatar'
+import Chip from '../components/Chip'
+import FaceIcon from '@material-ui/icons/Face'
+import { Playground, PropsTable } from 'docz'
+
+# Chip
+
+## Properties
+<PropsTable of={ Chip } />
+
+## Without Avatar
+<Playground>
+    <Chip label='Hello darkness my old friend' />
+</Playground>
+
+## With Initials Avatar
+<Playground>
+    <Chip
+        label='Hello darkness my old friend'
+        avatar={ <Avatar>HD</Avatar> }
+    />
+</Playground>
+
+## With Icon Avatar
+<Playground>
+    <Chip
+        label='Hello darkness my old friend'
+        avatar={ <Avatar><FaceIcon /></Avatar> }
+    />
+</Playground>
+
+## With Image Avatar
+<Playground>
+    <Chip
+        label='Hello darkness my old friend'
+        avatar={
+            <Avatar
+                src="https://www.google.com.br/images/branding/googlelogo/2x/googlelogo_color_272x92dp.png"
```

Comment:
i suggest to use another image because Google logo may have some "policy" on the use of their image